index.html

```diff
@@ -499,6 +507,11 @@ <h4>Cloud Code</h4>
                     <p>Guide</p>
                 </div>
             </a>
+            <a href="//docs.parseplatform.org/defined-schema/guide" target="_blank">
```

Comment:
Not sure why this feature has it's own guide. It's a feature of Parse Server, so I think it should be just a section within the [Parse Server guide](https://docs.parseplatform.org/parse-server/guide/) or Cloud Code guide.

Reply:
Because it was in https://docs.parseplatform.org/, I thought it should be added, if not, should I delete it?

Reply:
I don't think this adding the schema docs as a separate link is needed if we remove it later anyway.

I think we need to:

1. move the page with these guide boxes into a separate docs page
2. import that docs page on the front page
3. replace the page that is currently displayed at docs.parseplatform.org with with an import the new docs page.
4. conver the defined schema guide into a chapter of Parse Server guide or Cloud Code guide.

Not sure if you want to do it in this PR or we leave it for another PR.

Reply:
I think it is better to do these things in another PR

Reply:
I will removed the `Defined Schema` section in this PR